### DISCUSSION TITLE
Will github contact my school if I sign in for github student.

### DISCUSSION THREAD

**Discussion by anay-208** [community/community#57218] [GitHub Education] [answered]
### Hi everyone,

I m a 9th grader, who is thinking of applying to github student.
I've a question, will github contact my school relating it?
This will be a problem for me then. Thanks

**Comment by fsgreco** [comment, marked as answer]
No they don't, github makes sure you are an university student because you need to provide a university student email address - they check that and sometimes they asked you some kind of certificate (like the study plan in my case) but you have to provide it to them, they do not contact the university (as far as I know - also, I think it could be really tedious for github to contact the university every time somebody applies for github education program :D ).

**Comment by anay-208** [comment]
Thank you so much!
